packages/0x.js/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+v0.22.7 - _November 13, 2017_
```

Comment:
Since this changes the public interface, it should be a minor version bump: `0.23.0`